LABS-EU3/quality_hub_frontend
# Logout feature

##  What does this PR do
Add logout

##  Description of task to be completed
Add logout functionality to dashboard.

##  How should this manually be tested
log in, click on the profile icon in the dashboard then click logout,
check localstorage is empty and you are redirected to landing page

##  What are the relevant Trello board stories
As a user i want to be able to log out

##  Screenshots (If appropriate)


##  Questions (If any):
